README.md

```diff
@@ -6,6 +6,62 @@
 
 An SDK that build protocol logics for Protocolink
 
+## How to contribute?
+
+### Initial Setup
+
+1. Fork this repository.
+2. Set up your development environment and install the required dependencies.
+
+### Contribution Steps
+
+1. **Create a New Logic:**
+  - Add a new folder under `src/logics` named after the protocol (protocolId), using Hyphenation format.
+  - Create a new logic file under `src/logics/{protocolId}`, named with the action name (logicId). Prefix it with `logic.` and use Hyphenation format. The file name should be `logic.{logicId}.ts`.
+
+2. **Implement the Logic:**
+  - Implement the Logic class in the logic file using PascalCase. Append `Logic` to the action name.
+  - Extend the `core.Logic` class.
+  - Implement the following interfaces based on the requirements:
+    - `core.LogicTokenListInterface`: Support for tokens list function.
+    - `core.LogicOracleInterface`: Quotation function.
+    - `core.LogicBuilderInterface`: Txdata encode function for interacting with contracts.
```

Comment:
```suggestion
    - `core.LogicBuilderInterface`: Txdata encode functions for interacting with contracts.
```

Reply:
If there's only one function, should it be named in the plural form?

Reply:
Updated